### ISSUE TITLE
Logging via pandas DataFrame

### ISSUE THREAD

**Issue by schmittkl** [schmittkl/FARM#1]
Logging currently produces output via the standard logger, which is difficult to parse and via MLflow, which has a complex documentation structure. In order to get a quick overview over different experiments, a simple DataFrame that contains the evaluation data for each epoch/step would be sufficient.

So, I want to add the possibility to log the evaluation output to a third source, namely a pandas DataFrame which can then be stored on disk for every combination of epoch/step. I plan to do this via an extended version of the function log_results of the Evaluator class.

**Comment by schmittkl** [comment]
Solution can be found in [this branch](https://github.com/schmittkl/FARM/tree/pandas_logging).
For testing, you can use the files contained [here](https://github.com/schmittkl/Trapdoor/tree/main/MS1_FARM_features).

**Comment by rjschmitt** [comment]
is this also incorporated into master?

**Comment by schmittkl** [comment]
> is this also incorporated into master?

Yes, with [Pull Request #11](https://github.com/schmittkl/FARM/pull/11).
